src/test/java/edu/oakland/test/display02/DisplayComm2WayInterfaceImplementationTests.java

```diff
@@ -0,0 +1,18 @@
+import static org.junit.jupiter.api.Assertions.assertThrows;
```

Comment:
You need to make a method to test that the RFID method is passed in and a track data is passed, and checks that the status message of the TrackData is the output. That should finish your test coverage, and there are other examples of this merged into main if you look around in other Comm classes. Let me know if you have any other questions, and I'd be happy to help!